src/components/entity/list.vue

```diff
@@ -199,6 +209,25 @@ export default {
 
       this.$refs.table.afterUpdate(index);
     },
+    showResolve(index) {
+      if (this.refreshing) return;
+      this.updateIndex = index;
+      const odataEntity = this.odataEntities.value[index];
+      this.resolve.entity = odataEntity;
+      this.showModal('resolve');
+    },
+    hideResolve() {
+      this.hideModal('resolve');
+      this.resolve.entity = null;
+    },
+    afterResolve() {
+      const index = this.updateIndex;
+
+      const newOData = Object.assign(Object.create(null), this.odataEntities.value[index]);
+      newOData.__system.conflict = null;
```

Comment:
not sure if "Last updated" should be changed here

Reply:
Backend doesn't change `updatedAt` in case of resolve. I think it should, just like Submission's updatedAt is changed when reviewState is changed